cafebabel/static/js/tag-detail.js

```diff
@@ -1,35 +1,20 @@
 const container = document.querySelector('.tag-header')
-const imgUrl = container.getAttribute('data-image')
+const imgUrl = container.querySelector('img')
```

Comment:
It looks a bit too vague to me, why not `.tag-header img`?

Reply:
ok